transformer/classes/dockerfile/dockerfileparser.go

```diff
@@ -79,11 +79,11 @@ func (t *DockerfileParser) Transform(newArtifacts []transformertypes.Artifact, o
 		}
 		processedImages[sImageName.ImageName] = true
 		if paths, ok := a.Paths[artifacts.DockerfilePathType]; ok && len(paths) > 0 {
-			prjPath := filepath.Dir(paths[0])
-			if prjPaths, ok := a.Paths[artifacts.ProjectPathPathType]; ok && len(prjPaths) > 0 {
-				prjPath = prjPaths[0]
+			servicePath := filepath.Dir(paths[0])
```

Comment:
Would ServicePath be confusing for users, as to whether it is Dir Path or Url Path?